rclcpp/include/rclcpp/node_interfaces/node_graph_interface.hpp

```diff
@@ -146,7 +147,9 @@ class NodeGraphInterface
   /**
    * A topic is considered to exist when at least one publisher or subscriber
    * exists for it, whether they be local or remote to this process.
-   * The returned names are the actual names used and do not have remap rules applied.
+   * The returned names are the actual names after remap rules applied.
```

Comment:
I think I suggested this change because I got confused with the original wording, e.g.:
```
node->create_publisher("my_topic", ...)
auto result = another_node->get_topic_names_and_types();
```
The result will have an item `/first/node/namespace/my_topic`, and not simply `my_topic`.
But now I see that the original comment was about `another_node` not transforming the discovered topic name at all, which is also true (remapping rules are applied by `node`, not by `another_node`).

See also https://github.com/ros2/rclcpp/pull/1225, https://github.com/ros2/rclcpp/issues/1174.

Maybe we could reword this as:

```
The returned names are the actual names of the topics, either announced by another nodes or by this one.
No transformation is applied to the announced names by this node.
```

@sloretz what do you think? (I think you introduced https://github.com/ros2/rclcpp/pull/1225).

Reply:
@ivanpauno @sloretz thanks for the consideration, sounds good to me. i will update it accordingly.